Microsoft.Identity.Web/TokenCacheProviders/MsalAbstractTokenCacheProvider.cs

```diff
@@ -69,7 +69,7 @@ private string CacheKey
                 }
                 else
                 {
-                    return _httpContextAccessor.HttpContext.User.GetMsalAccountId();
+                    return _httpContextAccessor.HttpContext.GetUserTokenCacheKey();
```

Comment:
Just a reminder that when we apply this PR to the web-app tutorial, we need to adapt the code to use the new method `SetTokenCacheKey()`.

Reply:
I discussed with Mark. We need to do better.
cc: @MarkZuber